Document the fact that owfs has been patched to always read /etc/owfs.conf
--- a/src/man/man1/owfs.man
+++ b/src/man/man1/owfs.man
@@ -42,6 +42,11 @@
 .so man1/format.1so
 .so man1/job_control.1so
 .so man1/configuration.1so
++.PP
++On Debian,
++.B owfs (1)
++always reads
++.I /etc/owfs.conf
 .so man1/help.1so
 .so man1/timeout.1so
 .SH EXAMPLE
